aslam_cv_common/include/aslam/common/hamming-inl.h

```diff
@@ -41,9 +41,9 @@
 #ifndef ASLAM_COMMON_HAMMING_INL_H_
 #define ASLAM_COMMON_HAMMING_INL_H_
 
-#ifdef __ARM_NEON__
```

Comment:
could you ellaborate on why this change was necessary?

Reply:
__ARM_NEON__ is legacy and will only be set for `armv7` processors, whereas __ARM_NEON is set for both `armv7` and `armv8` processors (see [here](http://www.openwall.com/lists/john-dev/2016/01/18/1))